EpsilonAuxBmsV2/Core/Inc/Tasks/ContactorFunctions/CheckCurrent.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+#include "stm32f4xx_hal.h"
+
+extern ADC_HandleTypeDef hadc1;
+
+static const float CURRENT_SENSE_RESISTOR  = 0.001; // 1 mOhm
+static const int GAIN  = 250;
+static const float CURRENT_LOWER_THRESHOLD  = 0.260; // Lower current threshold (A)
```

Comment:
Why 0.26 again? isCurrentLow tripping?

Reply:
Nvm new relays, but is this value something electrical decided?

Reply:
I talked to Colton about it and he was ok with increasing it a little bit just in case